src/xmppjs/XJSAccount.ts

```diff
@@ -288,6 +299,7 @@ export class XmppJsAccount implements IBifrostAccount {
             `${components.room}@${components.server}/${components.handle}`,
         ));
         this.roomHandles.delete(room);
+        this.lastStanzaTs.delete(room);
```

Comment:
This is good; just wondering if this fixed something specific?  Or is it just general cleanup?

Reply:
This was fixing the case where we had left a room, but hadn't cleared the last stanza cache. I believe this was causing us to attempt (and fail) to rejoin the room because we thought we had forgotten about the room.